scripts/copper/bulk/prepare.py

```diff
@@ -16,19 +16,32 @@
 
 if __name__ == "__main__":
     bulk_copper = bulk("Cu", "fcc", 3.8)  #
-    data_path = Path("data/copper/bulk")
+    data_path_k = Path("data/copper/bulk_k")
+    data_path_ce = Path("data/copper/bulk_ce")
```

Comment:
only define this when it is needed (ie below k convergene code. either just call both of these varibles data_path or call it by the full name data_path_cutoff_energy. Only use acronyms when they are `standard` (it is not clear to me that ce stands for cutoff_energy)